proc/process.c

```diff
@@ -559,6 +560,10 @@ int process_load(process_t *process, vm_object_t *o, offs_t base, size_t size, v
 	hal_memset(reloc, 0, sizeof(reloc));
 
 	for (i = 0, j = 0, phdr = (void *)ehdr + ehdr->e_phoff; i < ehdr->e_phnum; i++, phdr++) {
+
+		if (phdr->p_type == PT_GNU_STACK)
+			gnustacksz = phdr->p_memsz;
```

Comment:
Align this size to page size, just in case.

Reply:
Done